lib/potassium/recipes/yarn.rb

```diff
@@ -0,0 +1,14 @@
+class Recipes::Yarn < Rails::AppBuilder
```

Comment:
Missing top-level class documentation comment.<br>Missing frozen string literal comment.<br>Use nested module/class definitions instead of compact style.